TestBinRucio.test_create_rule on master fails

Motivation
----------
```
________________________ TestBinRucio.test_create_rule _________________________
[gw0] linux -- Python 3.6.8 /usr/bin/python
test_bin_rucio.py:925: in test_create_rule
    assert not err
E   AssertionError: assert not '\x1b[31;1m2021-03-16 17:31:15,782\tERROR\tThere are not enough target RSEs to fulfil the request at this time.\nDetails: Target RSE set not sufficient for number of copies. (3 copies requested, RSE set size 1)\x1b[0m\n'
----------------------------- Captured stdout call -----------------------------
$> rucio upload --rse MOCK4 --scope data13_hip /tmp/file_SPHOYPINDA
 
$> rucio-admin rse add MOCK_UZTXIKFVRD
Added new deterministic RSE: MOCK_UZTXIKFVRD

$> rucio-admin rse set-attribute --rse MOCK_UZTXIKFVRD --key spacetoken --value ATLASSCRATCHDISK
Added new RSE attribute for MOCK_UZTXIKFVRD: spacetoken-ATLASSCRATCHDISK 
 
$> rucio-admin rse add MOCK_HHOXFJRRKR
Added new deterministic RSE: MOCK_HHOXFJRRKR
 
$> rucio-admin rse set-attribute --rse MOCK_HHOXFJRRKR --key spacetoken --value ATLASSCRATCHDISK
Added new RSE attribute for MOCK_HHOXFJRRKR: spacetoken-ATLASSCRATCHDISK 
 
$> rucio-admin rse add MOCK_RPSHBQSSLG
Added new deterministic RSE: MOCK_RPSHBQSSLG
 
$> rucio-admin rse set-attribute --rse MOCK_RPSHBQSSLG --key spacetoken --value ATLASSCRATCHDISK
Added new RSE attribute for MOCK_RPSHBQSSLG: spacetoken-ATLASSCRATCHDISK 
 
$> rucio add-rule data13_hip:file_SPHOYPINDA 3 'spacetoken=ATLASSCRATCHDISK'
```

--- a/lib/rucio/tests/test_bin_rucio.py
+++ b/lib/rucio/tests/test_bin_rucio.py
@@ -996,12 +996,12 @@ class TestBinRucio(unittest.TestCase):
         self.account_client.set_local_account_limit('root', tmp_rse, -1)
 
         # add rse atributes
-        cmd = 'rucio-admin rse set-attribute --rse {0} --key spacetoken --value ATLASSCRATCHDISK'.format(tmp_rse)
+        cmd = 'rucio-admin rse set-attribute --rse {0} --key spacetoken --value ATLASDELETERULE'.format(tmp_rse)
         print(self.marker + cmd)
         exitcode, out, err = execute(cmd)
         print(out, err)
         # add rules
-        cmd = "rucio add-rule {0}:{1} 1 'spacetoken=ATLASSCRATCHDISK'".format(self.user, tmp_file1[5:])
+        cmd = "rucio add-rule {0}:{1} 1 'spacetoken=ATLASDELETERULE'".format(self.user, tmp_file1[5:])
         print(self.marker + cmd)
         exitcode, out, err = execute(cmd)
         print(err)

